src/styles/main.scss

```diff
@@ -1,3 +1,57 @@
+$backroundColor: #eee;
+$cellSize: 100px;
+$cellGap: 1px;
+$calendarPadding: 10px;
+$cellFonts: "Arial", "Helvetica", sans-serif;
+
 body {
```

Comment:
[FROM README.MD] Display a calendar in the middle of the screen. 
Add flexbox to the body and align the content so it will be in the middle of the screen